customer-terminate-service/app/src/main/java/se/inera/intyg/cts/application/api/TerminationController.java

```diff
@@ -63,19 +63,19 @@ TerminationDTO startErase(@PathVariable UUID terminationId) {
 
   /**
    * Trigger a resend of the password
-   * @param terminationId
-   * @return
+   * @param terminationId termination id tied to the termination password that should be resent
```

Comment:
Maybe could rephrase the sentence? "termination id that should have its password resent"?